frontend/src/lib/error/index.ts

```diff
@@ -106,6 +108,13 @@ function setupGlobalErrorHandlers(error: Writable<App.Error | null>): void {
   function errorIsFromExtension(error: Error): boolean {
     return !!error.stack &&
       // tested on Chrome, Edge, Firefox and Brave
-      (error.stack.includes('chrome-extension://') || error.stack.includes('moz-extension://'));
+      (error.stack.includes('chrome-extension://') || error.stack.includes('moz-extension://')
+      || originatesFromAnonymousScript(error.stack));
+  }
+
+  function originatesFromAnonymousScript(stack: string): boolean {
```

Comment:
should we maybe disable this for local dev? basically, show anonymous errors for localdev?

Reply:
That sounds good 👍